### ISSUE TITLE
Less than 0 items for advanced start

### ISSUE THREAD

**Issue by Cooldude2606** [explosivegaming/scenario#126]
**Factorio version**
Version number: 0.17.66

**Scenario version**
Version number: 5.9.1

**Bug description**
Advanced start attempts to place an invalid amount of items into the players inventory.
```
Item stack count has to be a positive number
stack traceback:
    ...temp/currently-playing/modules/addons/advanced-start.lua:26: in function <...temp/currently-playing/modules/addons/advanced-start.lua:9>
```
**Reproduction method**
1. Unsure, sometimes when a player joins this error is created even with check for count being greater than 0

**Expected behavior**
No items given rather than error.

**Debug command**
```lua
/interface
local items = require('config.advanced_start').items
for item,callback in pairs(items) do
  local stats = player.force.item_production_statistics
  local made = stats.get_input_count(item)
  local success,count = pcall(callback,made,stats.get_input_count,player)
  output(item..' '..count)
end
```

**Comment by tovernaar123** [comment]
if the count is and float value it also says it shood be a postive number so that might be the problem

**Comment by Cooldude2606** [comment]
Did not think of that should be tested to see if something like >=1 would work better

**Comment by Cooldude2606** [comment]
Fixed in PR Fix Spawn Items Equal 0 #141
